spec/services/sufia/admin_set_create_service_spec.rb

```diff
@@ -2,7 +2,8 @@
 
 RSpec.describe Sufia::AdminSetCreateService do
   let(:admin_set) { AdminSet.new(title: ['test']) }
-  let(:service) { described_class.new(admin_set, user) }
+  let(:workflow_name) { nil }
+  let(:service) { described_class.new(admin_set, user, workflow_name) }
```

Comment:
@jeremyf which tests do you expect to fail?

(pushing change to use the constant now.)